ConvivaIntegration/components/conviva/AdTracking.brs

```diff
@@ -0,0 +1,63 @@
+function initAdTracking(player, livePass, session = invalid)
+  this = {}
+
+  this["_player"] = player
+  this["_livePass"] = livePass
+  this["_podIndex"] = 0
+  this["_session"] = session
+  this["_adType"] = ""
+
+  this["adTrackingMode"] = {
```

Comment:
Extract this to some public enum-function as we have already. That this is also usable for the customer when he initializes the integration.